spec/services/food_service.rb

```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+
+RSpec.describe FoodService do
+  it 'can return foods' do
+    user_query = 'sweet potato'
+    result = FoodService.get_food_details(user_query)
+
+    expect(result).to have_key(:foods)
+
+    expect(result[:foods].first).to have_key(:gtinUPC)
```

Comment:
This is a great start! I'd expect to see a few more assertions here about the presence of some other keys that will be needed in the PORO